site/app/controllers/forum/ForumController.php

```diff
@@ -485,7 +489,11 @@ public function alterPost($modifyType){
                 $this->core->addErrorMessage("You do not have permissions to do that.");
                 return;
         }
-        if($modifyType == 0) { //delete post or thread
+        if($this->core->getQueries()->isThreadLocked($_POST['edit_thread_id']) and $this->core->getUser()->getGroup() > 1 ){
+            $this->core->addErrorMessage("Thread is locked");
```

Comment:
If the sentence in line 394 ends with a period, then it's better if their error messages are the same.